.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2
 jobs:
   test:
     docker:
-      - image: omisegoimages/blockchain-base:1.6-otp20
+      - image: omisegoimages/elixir-omg-circleci:v1.8-20190129-02
```

Comment:
This still didn't get run https://circleci.com/workflow-run/b287be1c-dc8b-44bb-b8a3-c0befa255278